src/clj/runbld/main.clj

```diff
@@ -36,6 +37,12 @@
      ;; for tests when #'really-die is redefed
      msg*)))
 
+(defmacro try-catch [body]
```

Comment:
You need `[& body]` here, because there can be more than one form.

Also can we maybe call this `try-log` instead (and put it in the `io` namespace), something closer to what it's doing?